src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -440,6 +339,7 @@ describe('convertInclude', () => {
       type: 'include',
       model: {
         href: includeSupportingAmp,
+        index: -1,
```

Comment:
Should this include supporting amp be added to the page data, this would avoid the negative index which should not happen in production

Reply:
Could we add an explicit test that checks each include has the correct index, maybe at the `convertToOptimoBlocks` level?

Reply:
We could add a test case here using includes page data you have created in this PR: https://github.com/bbc/simorgh/blob/latest/src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/index.test.js